class TournamentsException(Exception):
    pass


class CommandError(TournamentsException):
    pass
